WoWsShipBuilder.Web/Dialogs/CustomAccelerationDataDialog.razor

```diff
@@ -164,4 +164,17 @@
         MudDialog.Close(DialogResult.Ok(container));
     }
 
+    private bool ValidateCustomParameters()
```

Comment:
The name is more clear about what the method does now, however, it does not properly reflect its usage